docs/docs/references/configuration/cli/trivy_image.md

```diff
@@ -36,6 +36,7 @@ trivy image [flags] IMAGE_NAME
 ```
       --cache-backend string              cache backend (e.g. redis://localhost:6379) (default "fs")
       --cache-ttl duration                cache TTL when using redis as cache backend
+      --cf-params strings                 specify paths to override the Cloudformation parameters files
```

Comment:
We may want to disable the flag for image scanning.

Reply:
that's right, I updated it here: https://github.com/aquasecurity/trivy/pull/5507/commits/8d8d834bc6c279a51e5c6962ad280502302587d9